stdlib/optparse.pyi

```diff
@@ -209,18 +209,18 @@ class OptionGroup(OptionContainer):
     def set_title(self, title: str) -> None: ...
 
 class Values:
-    def __init__(self, defaults: Mapping[str, Incomplete] | None = None) -> None: ...
-    def _update(self, dict: Mapping[str, Incomplete], mode) -> None: ...
-    def _update_careful(self, dict: Mapping[str, Incomplete]) -> None: ...
-    def _update_loose(self, dict: Mapping[str, Incomplete]) -> None: ...
-    def ensure_value(self, attr: str, value): ...
-    def read_file(self, filename: str, mode: str = "careful") -> None: ...
-    def read_module(self, modname: str, mode: str = "careful") -> None: ...
+    def __init__(self, defaults: Mapping[str, object] | None = None) -> None: ...
+    def _update(self, dict: Mapping[str, object], mode: Literal["careful", "loose"]) -> None: ...
+    def _update_careful(self, dict: Mapping[str, object]) -> None: ...
+    def _update_loose(self, dict: Mapping[str, object]) -> None: ...
+    def ensure_value(self, attr: str, value: object) -> Any: ...  # return type cannot be known statically
```

Comment:
Used `object` instead of `Any` since `Mapping` is covariant in its values type and the `Values` class doesn't typically get subclassed